stream_alert/rule_promotion/main.py

```diff
@@ -0,0 +1,20 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from stream_alert.rule_promotion.promoter import RulePromoter
```

Comment:
Don't forget the `from __future__ import absolute_import` or whatever it is to get rid of the logging error at the start of each invocation

Reply:
ah thanks!! good catch